tests/FakeItEasy.Tests/FakeItEasy.Tests.csproj

```diff
@@ -273,4 +272,4 @@
     </PropertyGroup>
     <Error Condition="!Exists('..\..\packages\StyleCop.MSBuild.4.7.49.1\build\StyleCop.MSBuild.Targets')" Text="$([System.String]::Format('$(ErrorText)', '..\..\packages\StyleCop.MSBuild.4.7.49.1\build\StyleCop.MSBuild.Targets'))" />
   </Target>
-</Project>
+</Project>
```

Comment:
VS doesn't respect `.editorconfig` when it plays with .csproj files. AFAIK the plug-in only respects files that pass through the text editor window.